routes/web.php

```diff
@@ -10,419 +9,372 @@
-        ->name('employees.select2_selection');
-
-    Route::resource('employees', '\App\Modules\Pim\Http\Controllers\EmployeesController', ['names' => [
-        'index' => 'employees.index',
-        'create' => 'employees.create',
-        'show' => 'employees.show',
-        'edit' => 'employees.edit',
-        'store' => 'employees.store',
-        'update' => 'employees.update',
-        'destroy' => 'employees.destroy'
-    ]]);
-
-    Route::get('candidates/datatable', '\App\Modules\Pim\Http\Controllers\CandidatesController@getDatatable')
-        ->name('candidates.datatable');
-
-    Route::resource('candidates', '\App\Modules\Pim\Http\Controllers\CandidatesController', ['names' => [
-        'index' => 'candidates.index',
-        'create' => 'candidates.create',
-        'show' => 'candidates.show',
-        'edit' => 'candidates.edit',
-        'store' => 'candidates.store',
-        'update' => 'candidates.update',
-        'destroy' => 'candidates.destroy'
-    ]]);
-
-    Route::group(['prefix' => 'profile/{employeeId}', 'as' => 'employees.'], function($employeeId) {
-        Route::resource('social-media', '\App\Modules\Pim\Http\Controllers\EmployeeSocialMediaController', ['names' => [
-            'index' => 'social_media.index',
-            'create' => 'social_media.create',
-            'show' => 'social_media.show',
-            'edit' => 'social_media.edit',
-            'store' => 'social_media.store',
-            'update' => 'social_media.update',
-            'destroy' => 'social_media.destroy'
-        ]]);  
-
-        Route::get('documents/datatable', '\App\Modules\Pim\Http\Controllers\EmployeeDocumentsController@getDatatable')
-        ->name('documents.datatable');
-
-        Route::get('documents/generate', '\App\Modules\Pim\Http\Controllers\EmployeeDocumentsController@generate')
-        ->name('documents.generate');
-
-        Route::post('documents/template-content', '\App\Modules\Pim\Http\Controllers\EmployeeDocumentsController@generateTemplateContent')
-        ->name('documents.template_content');
-        
-        Route::resource('documents', '\App\Modules\Pim\Http\Controllers\EmployeeDocumentsController', ['names' => [
-            'index' => 'documents.index',
-            'create' => 'documents.create',
-            'show' => 'documents.show',
-            'edit' => 'documents.edit',
-            'store' => 'documents.store',
-            'update' => 'documents.update',
-            'destroy' => 'documents.destroy'
-        ]]);  
-
-        Route::resource('contact-details', '\App\Modules\Pim\Http\Controllers\EmployeeContactDetailsController', [
-            'only' => [
-                'index',
-                'store',
-                'update'
-            ],
+Route::group(['prefix' => 'admin', 'middleware' => ['auth', 'admin']], function() {
+    Route::get('/', '\App\Http\Controllers\HomeController@index')->name('home');
+    Route::get('/storage/{path}', function($path) {
```

Comment:
The storage route should be available no matter the logged in user. Move it out of the Admin prefix. It can go at the bottom of the file, after the Auth part.